src/assets/tailwind.css

```
@@ -28,6 +28,20 @@ div[contenteditable='true']:focus {
   grid-template-rows: auto auto minmax(0, 1fr) auto minmax(0, 1fr);
 }
 
+.column-resizer {
+  position: relative;
+  z-index: 100;
+}
+
+.column-resizer:hover::after {
+  position: absolute;
```

Same as above comment, using tailwind  utilities makes things tidier.

I'm not entirely sure that `-left-6px` and `-right-6px` will work. If not you can keep the way you set it originally for the `left` and `right` properties.

```suggestion
  content: '';
  @apply absolute top-0 -left-6px -right-6px h-full; 
```

---

I'm struggling getting `content: ''` working, I was looking at this https://github.com/tailwindlabs/tailwindcss/issues/1306